UnitTests/BraintreePayPalTests/BTPayPalEditRequest_Tests.swift

```diff
@@ -0,0 +1,16 @@
+import XCTest
+@testable import BraintreeCore
+@testable import BraintreePayPal
+
+class BTPayPalEditRequest_Tests: XCTestCase {
+
+    override func setUp() {
+        super.setUp()
+    }
+
+    func testBTPayPalEditRequestInitializer() {
```

Comment:
We don't need to that setting a property works, we can trust the Swift lang to do this

Instead, I like your approach above and we can start adding functional test names to fill in later. I imagine something similar to [this](https://github.com/braintree/braintree_ios/blob/9f8543fecee17be445f5b7f9aaa8a6099bade66d/UnitTests/BraintreePayPalTests/BTPayPalVaultRequest_Tests.swift#L73), to verify that our JSON post body is properly formatted based on this edit request.